text/0000-add-status-field.md

```diff
@@ -0,0 +1,86 @@
+# Meta
+[meta]: #meta
+- Name: Add Status to RFC Metadata
+- Start Date: 2021-07-14
+- Author(s): @aemengo, @ekcasey
+- Status: Draft <!-- Acceptable values: Approved, On Hold, Draft -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This is a proposal to add a `Status:` field to the `Meta` section of the RFC template.
+
+# Definitions
+[definitions]: #definitions
+
+**Core team:** The party that votes on the approval of RFCs.
+
+# Motivation
+[motivation]: #motivation
+
+This RFC formally introduces the **"On Hold"** status for an RFC, in a way that's clear to the community.
+
+# What it is
+[what-it-is]: #what-it-is
+
+A new `Status:` field in the `Meta` section of the RFC template. See [#meta](#meta) for example.
+
+The acceptable values are distinguished in the following ways:
+- **Draft**: The Core team has not yet approved the RFC. Edits are still being made.
+- **Approved**: The Core team has approved the RFC and is accepting PRs for its implementation.
+- **On Hold**: The Core team has approved the RFC, but PRs are currently not being accepted for its implementation. A reason must be provided in the RFC.
+
+The `Status:` field does not intend to reflect states that are already implied by the RFC process. Examples including: *In Review*, *Closed*, etc.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+We will add the following to the RFC `Meta` section template:
+
+```
+- Status: Draft <!-- Acceptable values: Approved, On Hold, Draft -->
+```
+
+When an RFC is approved, the [../merge-rfc.sh](../merge-rfc.sh) script reflects the change like so:
+
+```
+- Status: Approved
+```
+
+When an RFC is put on hold (_via Pull Request_) then change is reflected like so:
+
+```
+- Status: On Hold
+```
+
```

Comment:
```suggestion

Pull requests that change the status of a previously accepted RFC to "On Hold", should add a Section titled `Notes` to the top of the RFC containing an explanation for the change of status. 

Example:
\```
# Notes

## On Hold
Since the approval of this RFC additional concerns X, Y, and Z have been raised. Solving problem Foo is still a priority for the project but the core team has decided to place this RFC on hold pending further discussion of alternative solutions. If you are interested in problem Foo place participate in the discussion on RFC #XXXX.
\```

If at a later date the core team or responsible sub team wishes to move the RFC from "On Hold" back to "Approved", this should be done via PR and another note should be added, describing the resolution of the discussion and the rationale for proceeding with the original RFC.
```

Reply:
I feel if we want to move a RFC back off of hold there should have a superseding RFC (in most cases).